contentpacks/khanacademy.py

```diff
@@ -425,7 +425,7 @@ def retrieve_exercise_dict(lang=None, force=False) -> str:
 
 
 @cache_file
-def download_and_clean_kalite_data(url, path) -> str:
+def download_and_clean_kalite_data(url, path, lang) -> str:
```

Comment:
Good catch, this worked because I happen to call this as a keyword argument. Amended my code to make it explicitly a keyword argument.